src/test/java/ru/sbt/mipt/oop/EventHandlersTest.java

```diff
@@ -0,0 +1,62 @@
+package ru.sbt.mipt.oop;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Objects;
+
+public class EventHandlersTest {
+    SmartHome smartHome;
+
+    @Before
+    public void init(){
+        this.smartHome = new JSONSmartHomeReader("smart-home-1.js").readHome();
+    }
+
+    @Test
+    public void HandleDoorOpenTest(){
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.